.github/workflows/ci.yaml

```diff
@@ -91,7 +99,7 @@ jobs:
             _build/test
             priv/plts
           key: ${{ runner.os }}-${{ env.OTP_VERSION }}-${{ env.ELIXIR_VERSION }}-${{ hashFiles('mix.lock') }}
-      - name: Check for Unused Dependencies
+      - name: Check for unused dependencies
```

Comment:
I don't really get this name change, as all the remaining step names are in title case

Reply:
Just copied from web :smile: Putting back as it was.